app/src/main/res/values/secret.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
-    <string name="google_api_key_android">INSERT YOUR GOOGLE API KEY HERE</string>
+    <string name="google_api_key_android">AIzaSyDU7WH2esnF2q2p4b1nYAZdKzqQmBfuKO4</string>
```

Comment:
You shouldn't commit the secret.xml file because we don't want to share any Google API key (especially the one we use internally). I would suggest making another commit "INSERT YOUR GOOGLE API KEY HERE" back in this file

Reply:
In the future, avoid using git add -A or git add . or git add * because it will add the secret.xml file even though it's in the .gitignore

Reply:
Yea this is bad :/ you should force overwrite this commit actually to get rid of this API key being in our commit logs.

Reply:
I don't *believe* this is our Berkeley Mobile API key (I think it's Sid's personal one) but either way I agree we should purge it ASAP and make sure this doesn't happen in the future

Reply:
I was talking to @rustielin about it yesterday and for some reason git add . (I thought git add . is supposed to ignore?) wasn't ignoring the files in the gitignore. I tried clearing the cache but that somehow led to those files being deleted in the commit. I added them back but I guess I must have forgotten to change back the API Key. @CourtneyBrousseau you're right this is my personal API key but I'll overwrite it right now.

Reply:
This is in an old commit - how would I remove this specific file from history?

Reply:
Got it - I merged all the commit changes into one commit, but now I can't really delete this conversation. It's showing the API Key that isn't even there anymore in the commit history. Might want to do something about this. I'll get to writing tests in the meantime.

Reply:
I think only I can resolve this since I made the original comment so I’ll do that now